src/Sarif.Viewer.VisualStudio/HashHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal class HashHelper
+    {
+        internal static string GenerateHash(byte[] data)
+        {
+            var hashFunction = new SHA256Managed();
```

Comment:
>SHA256Managed [](start = 35, length = 13)

add  using:
using var hassFun...

Reply:
done

---
In reply to: [601270348](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r601270348) [](ancestors = 601270348)